backend/logger/fileHandler.py

```diff
@@ -0,0 +1,80 @@
+import os
+import uuid
+import datetime
+import atexit
+
+
+class FileHandler:
+    """
+    In charge of handling files for the logger. This class will open, close,
+    and name files for the logger. Responsible for storing logs in safe place.
+    In addition, it will pass file handles to different programs.
+    """
+
+    def __init__(self, file_name):
+        self.file_name = file_name
+        self.full_name = file_name
+        self.file_handle = None
+
+        atexit.register(self.closeFile)
+
+    def createName(self):
```

Comment:
I think we should aim to standardize snake case in python (except class names) and camel case in JS. what do you think?

Reply:
Agreed, will make changes.

Reply:
yep, I think it should